### DISCUSSION TITLE
[Esame 17 luglio 2024] Esercizio 1

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#580] [Esami M-Z]
![es.1](https://github.com/user-attachments/assets/0c3f81c6-5546-4fde-bc18-21d35c77835d)

<details>
  <summary>soluzione del prof</summary>
  <br>
  <img src="https://github.com/user-attachments/assets/2ed07faa-69dc-4636-bb46-0f1050ada4dd" alt="soluzione"  />
    <img src="https://github.com/user-attachments/assets/28ebbbfe-0793-4e1d-ac3c-58919373b1a8" alt="soluzione"  />
</details>

**Comment by niccolo-fato** [comment]
<img width="1280" height="854" alt="image" src="https://github.com/user-attachments/assets/8c9e1bc5-193e-4050-9764-0e52e254f494" />
